test_runner/src/main/kotlin/ftl/args/yml/errors/ConfigurationErrorMessageBuilder.kt

```diff
@@ -0,0 +1,50 @@
+package ftl.args.yml.errors
+
+import com.fasterxml.jackson.databind.JsonNode
+import java.lang.Exception
+
+class ConfigurationErrorMessageBuilder {
```

Comment:
Minor suggestion. This class has no state and no constructor arguments, there is no reasons to create new instances. It could be object or even factory function.

Reply:
Thanks for this! I made changes :)